md/enhanced.py

```diff
@@ -351,3 +351,98 @@ def sample_from_log_weights(weighted_samples, log_weights, size):
     assert np.abs(np.sum(weights) - 1) < 1e-5
     idxs = np.random.choice(np.arange(len(weights)), size=size, p=weights)
     return weighted_samples[idxs]
+
+
+from fe import free_energy
+from md import minimizer
+from timemachine import lib
+from timemachine.lib import custom_ops
+from md import builders
+from md.barostat.utils import get_group_indices, get_bond_list
+
+
+def get_solvent_phase_system(mol, ff):
+    masses = np.array([a.GetMass() for a in mol.GetAtoms()])
+    water_system, water_coords, water_box, water_topology = builders.build_water_system(3.0)
+    water_box = water_box + np.eye(3) * 0.5  # add a small margin around the box for stability
```

Comment:
I thought minimization removed our need for the margin, are there still cases that we need it? Or is stability specific to the enhanced sampling?

Reply:
I think there are still cases where waters get added at the PBC boundary - the barostat will equilibrate the box size down to the correct size though.